Products/ZenEvents/zentrap.py

```diff
@@ -39,26 +39,32 @@
 from Products.ZenHub.interfaces import ICollectorEventTransformer
 
 from Products.ZenCollector.daemon import CollectorDaemon
-from Products.ZenCollector.interfaces import ICollector, ICollectorPreferences, \
-    IEventService, \
-    IScheduledTask, IStatisticsService
-from Products.ZenCollector.tasks import SimpleTaskFactory,\
-                                        SimpleTaskSplitter,\
-                                        BaseTask, TaskStates
+from Products.ZenCollector.interfaces import (
+    ICollector,
+    ICollectorPreferences,
+    IEventService,
+    IScheduledTask,
+    IStatisticsService
+)
+from Products.ZenCollector.tasks import (
+    SimpleTaskFactory,
+    SimpleTaskSplitter,
+    BaseTask,
+    TaskStates
+)
 from Products.ZenUtils.observable import ObservableMixin
 
 
 from pynetsnmp import netsnmp, twistedsnmp
 
 from Products.ZenUtils.captureReplay import CaptureReplay
 from Products.ZenEvents.EventServer import Stats
-from Products.ZenUtils.Utils import unused
 from Products.ZenEvents.TrapFilter import TrapFilter, TrapFilterError
 from Products.ZenEvents.ZenEventClasses import Clear, Critical, Error
 from Products.ZenUtils.Utils import unused, zenPath
 from Products.ZenCollector.services.config import DeviceProxy
 from Products.ZenHub.services.SnmpTrapConfig import User
-unused(Globals, DeviceProxy, User)
+unused(Globals, DeviceProxy, User, Error, zenPath)
```

Comment:
Generally, `unused` is for imports that are necessary for their side-effects.  I know `zenPath` has no side-effects and so its import can be removed.  You'll have to test whether `Error` is the same.